### PR TITLE
 show RBF status for confirmed tx, remove RBF ON/OFF

### PR DESCRIPTION
**Pull request by slowbackspace**
SENT PENDING RBF=ON
Screenshot 2021-02-04 at 13 57 01

CONFIRMED
Screenshot 2021-02-04 at 13 52 18

RECV PENDING RBF=ON
Screenshot 2021-02-04 at 13 53 35

RECV PENDING RBF=OFF
Screenshot 2021-02-04 at 13 53 41

### CODE REVIEW

File: packages/suite/src/utils/wallet/transactionUtils.ts [hunk 1]
```diff
@@ -679,3 +679,8 @@ export const getBlockExplorerUrl = (tx: WalletAccountTransaction) => {
     const network = getNetwork(tx.symbol);
     return `${network!.explorer.tx}${tx.txid}`;
 };
+
+export const isTxFinal = (tx: WalletAccountTransaction, confirmations: number) => {
+    // checks RBF status
+    return !!(!tx.rbf || confirmations > 0);
```
Comment (keraf) [inline packages/suite/src/utils/wallet/transactionUtils.ts]: Is it necessary to cast it into a boolean here? Both, `!tx.rbf` and `confirmations > 0`, should return bools already. And I guess for such a short function, the return statement can be omitted as well.

**Comment by keraf** [comment]
Apart from that single comment I left, the rest looks good!

File: packages/suite/src/utils/wallet/transactionUtils.ts (hunk 1 above)
Comment (slowbackspace) [reply packages/suite/src/utils/wallet/transactionUtils.ts]: that what eslint did by itself :D I can't place the comment inside if I omit the return statement 🤔

Comment (slowbackspace) [reply packages/suite/src/utils/wallet/transactionUtils.ts]: ok, removed !!, looks like now eslint is fine with it 🤷‍♂️

**Review by keraf — APPROVED** [review]

**Review by tomexx — APPROVED** [review]

**Comment by tsusanka** [comment]
@szymonlesisz please have a look at this "post-mortem" on Monday, I am merging now because we want to pass it to QA as soon as possible.

**Pull request merged**